### PR TITLE
Replaced deprecated LArray class by the new Array class.

### PR DESCRIPTION
**Pull request by alixdamman**
And use ``import larray as la`` everywhere.

### CODE REVIEW

File: larray_editor/comparator.py [hunk 1]
```diff
@@ -133,11 +134,11 @@ def update_isequal(self):
                 # scale reldiff to range 0-1 with 0.5 for reldiff = 0
                 self.bg_value = (reldiff / maxabsreldiff) / 2 + 0.5
             else:
-                self.bg_value = full_like(self.array, 0.5)
+                self.bg_value = la.full_like(self.array, 0.5)
         except TypeError:
             # str/object array
-            maxabsreldiff = nan
-            self.bg_value = full_like(self.array, 0.5)
+            maxabsreldiff = np.nan
```
Comment (gdementen) [inline larray_editor/comparator.py]: use la.nan please :)

File: larray_editor/comparator.py [hunk 2]
```diff
@@ -268,7 +269,7 @@ def _setup_and_check(self, widget, data, title, readonly, **kwargs):
         self.listwidget.setCurrentRow(0)
 
     def get_arrays(self, name):
-        return [aslarray(s.get(name, nan)) for s in self.sessions]
+        return [la.asarray(s.get(name, np.nan)) for s in self.sessions]
```
Comment (gdementen) [inline larray_editor/comparator.py]: idem

File: larray_editor/tests/test_api_larray.py [hunk 3]
```diff
@@ -73,51 +72,51 @@
...
 demo = make_demo(9, 2.5, 1.5)
 sphere = make_sphere(9, 4)
-extreme_array = LArray([-inf, -1, 0, nan, 1, inf])
-scalar = LArray(0)
-arr_empty = LArray([])
-arr_obj = ndtest((2, 3)).astype(object)
-arr_str = ndtest((2, 3)).astype(str)
-big = ndtest((1000, 1000, 500))
-big1d = ndtest(1000000)
+extreme_array = la.Array([-np.inf, -1, 0, np.nan, 1, np.inf])
```
Comment (gdementen) [inline larray_editor/tests/test_api_larray.py]: take those from la instead, please

File: larray_editor/tests/test_api_larray.py [hunk 4]
```diff
@@ -157,17 +156,17 @@ def make_demo(width=20, ball_radius=5, path_radius=5, steps=30):
...
+arr1 = la.ones((geo, sex))
+arr2 = la.random.normal(axes=(geo, sex))
 compare(arr1, arr2, atol=0.5)
 compare(arr1, arr2, rtol=0.3)
 
-arr2 = where(arr2 > 1, arr1, -arr1)
-arr1['M'] = nan
-arr2['M'] = nan
+arr2 = la.where(arr2 > 1, arr1, -arr1)
+arr1['M'] = np.nan
```
Comment (gdementen) [inline larray_editor/tests/test_api_larray.py]: idem

File: larray_editor/comparator.py (hunk 1 above)
Comment (alixdamman) [reply larray_editor/comparator.py]: What's the difference?

Comment (gdementen) [reply larray_editor/comparator.py]: None at this point, but it could in the future. Since we provide it in the larray namespace, it's better to use it.

**Comment by gdementen** [comment]
ready to squash & merge

**Pull request merged**